### PR TITLE
Zw378

### PR DESCRIPTION
**Pull request by ZrybeaWhitfield**
Added help prompt

### CODE REVIEW

File: src/com/shapeshifters/thecrash/controller/TheCrashApp.java [hunk 1]
```diff
@@ -69,6 +68,10 @@ public void execute() {
                         case "quit":
                             setGameOver(true);
                             break;
+                        case "i":
+                            viewInfo();
+                            promptEnterKey();
```
Comment (Alan-SDE) [inline src/com/shapeshifters/thecrash/controller/TheCrashApp.java]: I would suggest moving the promptEnterKey(); to the viewInfo() method

**Review by Alan-SDE — CHANGES_REQUESTED** [review]
move promptEnter to the viewInfo method

File: src/com/shapeshifters/thecrash/controller/TheCrashApp.java (hunk 1 above)
Comment (ZrybeaWhitfield) [reply src/com/shapeshifters/thecrash/controller/TheCrashApp.java]: On it

Comment (ZrybeaWhitfield) [reply src/com/shapeshifters/thecrash/controller/TheCrashApp.java]: Complete, ty

**Review by MandoGuardado — APPROVED** [review]

**Pull request merged**